modules/cnf-configuring-workload-hints.adoc

```diff
@@ -0,0 +1,84 @@
+
+The environment in which an application is operating influences its behavior. For a typical data center with no strict latency requirements, only minimal default tuning is needed that enables CPU partitioning for some high performance workload pods.
+For data centers and workloads where latency is a higher priority, measures are still taken to optimize power consumption.
+The most complicated cases are clusters close to latency-sensitive equipment such as manufacturing machinery and software-defined radios. This last class of deployment is often referred to as Far edge. For Far edge deployments, ultra-low latency is the ultimate priority, and is achieved at the expense of power management.
+
+[NOTE]
+====
+The following workload hints can be configured manually or created using the Performance Profile Creator. See "Creating a performance profile" for further information.
+====
+
+[cols="1,1,1,1",options="header"]
+|===
+    | Performance Profile creator setting| Hint | Environment | Description
+
+    | Default
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: false
+realtime: false
+----
+    | High throughput cluster without latency requirements
+    | Performance achieved through CPU partitioning only.
+
+
+
+    | Low-latency
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: false
+realtime: true
+----
+    | Regional datacenters
+    | Both energy savings and low-latency are desirable: compromise between power management, latency and throughput.
+
+
+    | Ultra-low-latency
+    a|[source,terminal]
+----
+workloadHints:
+highPowerConsumption: true
+realtime: true
+----
+    | Far edge clusters, latency critical workloads
+    | Optimized for absolute minimal latency and maximum determinism at the cost of increased power consumption.
+
+|===
+
+.Procedure
+
+. Create a `PerformanceProfile` appropriate for the environment's hardware and topology as described in previous sections. Then adjust the profile to match the expected workload. In this example, we tune for the lowest possible latency.
+
+. Add the `highPowerConsumption` and `realtime` workload hints and set them both to true as indicated by the above table.
++
+[source,yaml]
+----
+    apiVersion: performance.openshift.io/v2
+    kind: PerformanceProfile
+    metadata:
+      name: workload-hints
+    spec:
+      ...
+      workloadHints:
+        highPowerConsumption: true <1>
+        realtime: true <2>
+----
+<1> If `highPowerConsumption` is true, the node is tuned for very low latency at the cost of increased power consumption.
+<2> Disables some debugging and monitoring features that can affect system latency.
```

Comment:
I think this module needs to be restructured or broken up into multiple modules. The module begins as a concept. Then, it provides a table of reference materials. Finally, it has a procedure. 

I would also add an ==Additional resources section after the procedure with a link to "Creating a performance profile," because the first step of the procedure says "as described in previous sections", and it isn't in a previous section. It's actually in the subsequent assembly.

Let me know if you need help with this.

Reply:
As mentioned above, I relocated the conceptual material to the top level conceptual topic on low latency ("Understanding low latency"). I gave the table a topic of its own - it needs to precede the procedure, as you have to use the table to know the right workload hint settings to go for.